src/lib/inactivity.js

```diff
@@ -0,0 +1,16 @@
+export const inactivityTime = () => {
+  let time;
+  window.onload = resetTimer;
+  document.onmousemove = resetTimer;
+
+  function logout() {
```

Comment:
Suggestion: "logout" refers to an auth vocabulary, which is not the case here. I'd suggest a name that better reflects the action of the function, such as "goHome"

Reply:
goHome or redirection ?

Reply:
"goHome" is an action word and clearly indicate what the function do : it redirects to Home
"redirection" is not an action word and don't give so much information : were do the function redirect ?